general/cisetup/pipelinesyaml.go

```diff
@@ -0,0 +1,351 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-cli-core/utils/coreutils"
+	"github.com/jfrog/jfrog-cli/utils/cliutils"
+	"github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"gopkg.in/yaml.v2"
+	"strconv"
+	"strings"
+)
+
```

Comment:
I suggest we orginize the code in this file inside a struct named JFrogPipelinesYamlGenerator. This struct can have one public entry point named Generate.

Let's do the same for the code inside ```pipelinesrun```. We can perhaps name the struct there ```JFrogPipelinesConfigurator```.

Both structs should move to jfrog-cli-core IMO.